packages/fxa-settings/src/pages/Legal/en.ftl

```diff
@@ -0,0 +1,7 @@
+## Legal page
+
+legal-header = Legal
```

Comment:
Could you add a comment that explains what kind of content follows? "Legal" by itself could be a bit vague and hard to translate, so some more context might allow for better translations.

Reply:
@bcolsson How's it looking with the comment at the top of this file now? There's not much context unfortunately, it's just a single header with some links (see https://accounts.firefox.com/legal)

Reply:
Thanks for adding the comment @LZoog , it should help!